demo/docbot/ingestion.py

```diff
@@ -8,7 +9,20 @@
 # check out the _bulk python client helper.
 
 # Documents should be ingested from the /data/ dir
+DEMOS_PATH = os.path.abspath(os.path.join(__file__,  "..", "..",".."))
+DOCUMENTATION_INDEX_PATH = "data/documentation-index.json"
+WEBSITE_INDEX_PATH = "data/website-index.json"
+
+def read_files_from_data() -> (list, list):
```

Comment:
Overall I really like the path you are going on here. I recommend a few tweaks to make it a bit more flexible. Let's read the file list from the `data` directory and then iterate over it. That way we can add as many other sources in the future we want without having to modify the code.